code/game/machinery/dna_infuser/organ_sets/roach_organs.dm

```diff
@@ -0,0 +1,226 @@
+#define ROACH_ORGAN_COLOR "#7c4200"
+// Yeah i'm lazy and we don't use any of the other color slots
+#define ROACH_COLORS ROACH_ORGAN_COLOR + ROACH_ORGAN_COLOR + ROACH_ORGAN_COLOR
+
+/datum/armor/roach_internal_armor
+	bomb = 100
+	bio = 90
+
+/datum/status_effect/organ_set_bonus/roach
+	id = "organ_set_bonus_roach"
+	organs_needed = 4
+	bonus_activate_text = span_notice("Roach DNA is deeply infused with you! \
```

Comment:
Too late now but why do we make the DNA the subject for all of these instead of `you are deeply infused with x DNA`? Reads kind of weird.

Reply:
I don't know the real reason but, maybe to sound more sci-fi-y?